api/src/tooltool_api/static/tcauth.js

```diff
@@ -90,16 +95,8 @@ window.tcauth.setup = function(service, default_service_url) {
     $login.on('click', function(e) {
         e.preventDefault();
         e.stopPropagation();
-        var url = $('body').attr('data-treestatus-api-url') || 'https://localhost:8000';
-        $.ajax({
-            url: url + '/auth0/login',
-            error: function(xhr, status, error) {
-            },
-            success: function(redirect_url, status, xhr) {
-                window.location = redirect_url;
-            }
-
-        });
+        var url = config.taskclusterRootUrl + '/login/oauth/authorize?client_id=' + config.clientId + '&releng-tooltool-localdev&redirect_uri=' + config.redirectUri + '&response_type=code&scope=project%3Areleng%3Aservices%2Ftooltool%2F*&state=5&expires=5%20minutes';
```

Comment:
`releng-tooltool-localdev` is hardcoded, you probably forgot to remove it. `scope` looks like a good candidate to move to the configs.

Reply:
Oops, yes! It's not breaking anything, but it doesn't need to be there.